.github/workflows/livestream-docker-image.yml

```
@@ -0,0 +1,88 @@
+name: Livestream Docker CI/CD
+
+on:
+    push:
+        branches:
+            - master
+        paths:
+            - 'livestream/**'
+            - '.github/workflows/livestream-docker-image.yml'
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        permissions:
+            contents: read
+            packages: write
+
+        outputs:
+            sha: ${{ steps.push.outputs.digest }}
+
+        steps:
+            - name: Check out livestream code
+              uses: actions/checkout@v4
+              with:
+                  sparse-checkout: 'livestream/'
+                  sparse-checkout-cone-mode: false
+
+            - name: Log in to the Container registry
+              uses: docker/login-action@v3
+              with:
+                  registry: ghcr.io
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+                  logout: false
+
+            - name: Extract metadata (tags, labels) for Docker
+              id: meta
+              uses: docker/metadata-action@v5
+              with:
+                  images: ghcr.io/posthog/posthog/livestream
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v2
+
+            - name: Build and push Docker image
+              id: push
+              if: github.ref != 'refs/heads/master'
```

gotta revert back now :P 

```suggestion
              if: github.ref == 'refs/heads/master'
```